From 8d16cbc7d83c4eab20390217f18a7ac3f3b05ec0 Mon Sep 17 00:00:00 2001
From: Shreenidhi Shedi <sshedi@vmware.com>
Date: Tue, 22 Mar 2022 20:32:32 +0530
Subject: [PATCH] allow proc_security_t read permission to systemd_sysctl_t

This fixes the below issue during system boot.
Mar 22 12:55:19 photon-machine audit[481]: AVC avc:  denied  { read } for  pid=481 comm="systemd-sysctl" name="suid_dumpable" dev="proc" ino=13224 scontext=system_u:system_r:systemd_sysctl_t:s0 tcontext=system_u:object_r:proc_security_t:s0 tclass=file permissive=0

Signed-off-by: Shreenidhi Shedi <sshedi@vmware.com>
---
 policy/modules/system/systemd.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/system/systemd.te b/policy/modules/system/systemd.te
index db7cd71..0ec330d 100644
--- a/policy/modules/system/systemd.te
+++ b/policy/modules/system/systemd.te
@@ -1008,6 +1008,7 @@ kernel_dgram_send(systemd_sysctl_t)
 kernel_request_load_module(systemd_sysctl_t)
 kernel_rw_all_sysctls(systemd_sysctl_t)
 kernel_write_security_state(systemd_sysctl_t)
+kernel_read_security_state(systemd_sysctl_t)
 
 files_read_system_conf_files(systemd_sysctl_t)
 
-- 
2.25.1

